### ISSUE TITLE
twisted.python.release.BuildAPIDocsScript points to the wrong source URL

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8568]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8568|
|Type|defect|
|Created|2016-07-06 14:53:17Z|
|Branch|https://github.com/twisted/twisted/tree/docs-source-link-8568|

Fix:

```
diff --git a/twisted/python/_release.py b/twisted/python/_release.py
index 5f86fa0..2bf3da3 100644
--- a/twisted/python/_release.py
+++ b/twisted/python/_release.py
@@ -1033,7 +1033,7 @@ class BuildAPIDocsScript(object):
         """
         version = Project(projectRoot.child("twisted")).getVersion()
         versionString = version.base()
-        sourceURL = ("http://twistedmatrix.com/trac/browser/tags/releases/"
+        sourceURL = ("https://github.com/twisted/twisted/tree/"
                      "twisted-%s" % (versionString,))
         apiBuilder = APIBuilder()
         apiBuilder.build(
```

<details><summary>Searchable metadata</summary>

```
trac-id__8568 8568
type__defect defect
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_3_1 Twisted 16.3.1
branch__docs_source_link_8568 docs-source-link-8568
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467816797605488 1467816797605488
changetime__1469201088733522 1469201088733522
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

Working on this...

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

ready for review

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

PR at https://github.com/twisted/twisted/pull/384

Changes look good. Happy to see pydoctor 16.2.0 released.

Maybe you could add apidoc-builder in travis-CI together with this branch... in this way it will also fix [#8518](https://github.com/twisted/twisted/issues/8518)

please consider enabling the apidocs builder on travis-ci and then merge.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 49e548c3db53db38c28f0e09e72a8dde39e05275
```
#!CommitTicketReference repository="" revision="49e548c3db53db38c28f0e09e72a8dde39e05275"
Merge pull request #384 from twisted/docs-source-link-8568

Author: hawkowl
Reviewer: adiroiban
Fixes: #8568, #8518
```
